gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -145,8 +145,10 @@ spec:
           - {{ $gateway.envoyMetricsService.host }}:{{ $gateway.envoyMetricsService.port }}
         {{- end }}
         {{- if .Values.global.proxy.envoyAccessLogService.enabled }}
-          - --envoyAccessLogServiceAddress
-          - {{ .Values.global.proxy.envoyAccessLogService.host }}:{{ .Values.global.proxy.envoyAccessLogService.port }}
+          - --envoyAccessLogService
+          {{- with  .Values.global.proxy.envoyAccessLogService }}
```

Comment:
This seems far too much template... 

Can we just change the setting to include the expected string ? If that's the syntax of the flag, let's just let user use whatever envoy documents and not attempt to invent our own syntax ( and associated template complexity )